platforms/core-execution/execution/src/main/java/org/gradle/internal/execution/FileCollectionFingerprinter.java

```diff
@@ -29,11 +28,6 @@ public interface FileCollectionFingerprinter {
      */
     FileNormalizer getNormalizer();
 
-    /**
-     * Creates a fingerprint of the contents of the given collection.
-     */
-    CurrentFileCollectionFingerprint fingerprint(FileCollection files);
-
```

Comment:
💭 I've separated `FileCollectionFingerprinter` and `FileCollectionSnapshotter` more completely. Now they have distinct responsibilities:

- `FileCollectionSnapshotter` takes a `FileCollection` and returns a `FileSystemSnapshot`.
- `FileCollectionFingerprinter` takes a `FileSystemSnapshot` and turns it into a fingerprint.